src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/services.po.ts

```diff
@@ -59,6 +59,23 @@ export class ServicesPageHelper extends PageHelper {
           cy.get('#count').type(count);
           break;
 
+        case 'snmp-gateway':
+          this.selectOption('snmp_version', snmpVersion);
+          cy.get('#snmp_destination').type('192.168.0.1:8443');
```

Comment:
can we randomly specify an IP for snmp_destination? Or am I wrong in assuming this is by a random choice?

Reply:
Well it should be actually the IP address of a client  that receives the SNMP traps, but as long as service is running state I avoided caring abt it much